src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -798,4 +798,39 @@
   <data name="Profile_DeleteConfirmationMessage.Text" xml:space="preserve">
     <value>Are you sure you want to delete this profile?</value>
   </data>
+  <data name="Globals_TabSwitcherMode.Header" xml:space="preserve">
+    <value>Tab switcher mode</value>
+  </data>
+  <data name="Globals_TabSwitcherMode.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Defines the terminal behavior when switching tabs. When set to "Most recently used", tabs will be selected based on their most recently used ordering. When set to "In order", tabs will be switched in their current order on the tab bar. When set to "Disabled", the tab switcher is disabled.</value>
+    <comment>"Mru", "InOrder", and "Disabled" must match Globals_TabSwitcherModeMru.Content, Globals_TabSwitcherModeInOrder.Content, and Globals_TabSwitcherModeDisabled.Content values respectively.</comment>
+  </data>
+  <data name="Globals_TabSwitcherModeMru.Content" xml:space="preserve">
+    <value>Most recently used</value>
+  </data>
+  <data name="Globals_TabSwitcherModeInOrder.Content" xml:space="preserve">
+    <value>In order</value>
+  </data>
+  <data name="Globals_TabSwitcherModeDisabled.Content" xml:space="preserve">
+    <value>Disabled</value>

```

Comment:
"Do not use tab switching interface" might be more accurate an help our users. Otherwise, they may think "i cannot switch tabs"

Or, maybe we can change the whole thing. @cinnamon-msft:

Tab switcher interface style
* Separate window, in most-recently-used order
* Separate window, in tab strip order
* Traditional navigation, no separate window

what do you think? wordsmith it a bit.

Reply:
I like Dustin's recommendations here, I think they describe it well :)

Reply:
Although I would change it to "most recently used," removing the dashes